src/config/scrivitoSites.ts

```diff
@@ -86,7 +86,10 @@ export function isNoSitePresent(): boolean {
 export async function ensureSiteIsPresent() {
   if ((await load(currentSiteId)) === null) {
     if (await load(isNoSitePresent)) {
-      ensureUserIsLoggedIn()
+      // TODO: remove window check after #11101 is implemented
+      if (typeof window !== 'undefined') {
+        ensureUserIsLoggedIn()
+      }
```

Comment:
* [ ] We don't need this change.
  1. `ensureUserIsLoggedIn()` will still throw after #11101. However,
  2. This block is not in the prerendering code path